### PR TITLE
Update tap13.md

### PR DESCRIPTION
**Pull request by JustinCappos**
minor clarification suggested

### CODE REVIEW

File: tap13.md [hunk 1]
```diff
@@ -147,15 +147,16 @@ There likely also needs to be a clarity pass throughout to make this potential
...
-touch the clients, then the client may perhaps use a threshold of offline keys before delegating to
-a developer’s key.  [TAP 8](tap8.md) also provides support for cases where the key need to be rotated
-or changed and the key is still accessible to the developer.
+remedied before by the repository but this no longer works in every case.  For example,
+previously if the repository delegated to a target from the top-level targets role, that 
+file could be updated by the top-level targets role if Alice’s key changed or was lost.  
+However, as the repository’s root role is no longer trusted to provide top-level targets keys
+and different clients may have different top-level targets keys, any clients using this
+TAP must take more care.  Thus, one should take into account the operational difficultly to touch
+clients in the case of key loss or compromise for the top-level targets file.  If it is
```
Comment (mnm678) [inline tap13.md:156]:
```suggestion
clients in the case of key loss or compromise for the top-level targets metadata file.  If it is 
```

File: tap13.md [hunk 2]
```diff
@@ -147,15 +147,16 @@ There likely also needs to be a clarity pass throughout to make this potential
...
+remedied before by the repository but this no longer works in every case.  For example,
+previously if the repository delegated to a target from the top-level targets role, that 
+file could be updated by the top-level targets role if Alice’s key changed or was lost.  
+However, as the repository’s root role is no longer trusted to provide top-level targets keys
+and different clients may have different top-level targets keys, any clients using this
+TAP must take more care.  Thus, one should take into account the operational difficultly to touch
+clients in the case of key loss or compromise for the top-level targets file.  If it is 
+operationally difficult to touch the clients, then the client may perhaps use a threshold of 
+offline keys before delegating to a developer’s key.  [TAP 8](tap8.md) also provides support for 
+cases where the key need to be rotated or changed and the key is still accessible to the developer.
```
Comment (mnm678) [inline tap13.md:159]:
```suggestion
cases where the key needs to be rotated or changed and the key is still accessible to the developer.
```

**Review by mnm678 — APPROVED** [review]
A couple of nits, but LGTM

**Review by joshuagl — APPROVED** [review]
Thanks, this is a nice clarification. @mnm678's suggestions seem like good refinements before we merge.

**Review by joshuagl — APPROVED** [review]

**Review by mnm678 — APPROVED** [review]

**Pull request merged**